### PR TITLE
Deprecate Left and Right InnerJoins in favor of one InnerJoin.

### PR DESCRIPTION
**Pull request by duckbrain**
Potential fix for #274

### CODE REVIEW

**Comment by u007** [comment]
sorry, there are differences between left and right inner Join, this seems to force the join to become inner join

**Comment by u007** [comment]
i do not support this merge request

**Comment by stanislas-m** [comment]
@u007 The point of #274 issue is that `LEFT` or `RIGHT` `INNER` join doesn't exist, but `LEFT` or `RIGHT` `OUTER` JOIN does exists. :)

**Comment by u007** [comment]
aah, sorry, i didnt realise i didn't use it.
guess there there are only left/right join, inner join and full join
make sense :)

File: query_joins.go [hunk 1]
```diff
@@ -52,22 +52,26 @@ func (q *Query) RightOuterJoin(table string, on string, args ...interface{}) *Qu
 	return q
 }
 
-// LeftInnerJoin will append a LEFT INNER JOIN clause to the query
+// LeftInnerJoin will append an INNER JOIN clause to the query
+//
+// Deprecated: Use InnerJoin instead
 func (q *Query) LeftInnerJoin(table string, on string, args ...interface{}) *Query {
-	if q.RawSQL.Fragment != "" {
```
Comment (u007) [inline query_joins.go]: hi, can you help to add deprecation warning as well?

File: query_joins.go [hunk 2]
```diff
@@ -52,22 +52,26 @@ func (q *Query) RightOuterJoin(table string, on string, args ...interface{}) *Qu
...
-	return q
+	return q.InnerJoin(table, on, args...)
 }
 
-// RightInnerJoin will append a RIGHT INNER JOIN clause to the query
+// RightInnerJoin will append an INNER JOIN clause to the query
+//
+// Deprecated: Use InnerJoin instead
 func (q *Query) RightInnerJoin(table string, on string, args ...interface{}) *Query {
+	return q.InnerJoin(table, on, args...)
```
Comment (u007) [inline query_joins.go]: same deprecation warning here :)

**Pull request merged**